src/redux/wallets/state.ts

```diff
@@ -5,6 +5,7 @@ import BigNumber from 'bignumber.js';
 export interface IWalletState {
     id: string;
     name: string;
+    deviceId?: string;
```

Comment:
we need more details here: `deviceVendor` and `deviceModel`, based on this we will know how to communicate with the device